### PR TITLE
Remove deprecated ssl setting

### PR DESCRIPTION
**Pull request by donoghuc**
Remove the deprecated `ssl` config option and document how to replace it with `ssl_enabled`. Prep the 2.0.0 release.

### CODE REVIEW

**Review by robbavey — APPROVED** [review]
LGTM - Over to @karenzone for doc review

File: docs/index.asciidoc [hunk 1]
```diff
@@ -355,6 +343,18 @@ When <<plugins-{type}s-{plugin}-ssl_client_authentication>> causes a client to p
...
+
++WARNING: As of version `2.0.0` of this plugin, some configuration options have been replaced.
++The plugin will fail to start if it contains any of these obsolete options.
+
++[cols="<,<",options="header",]
+ |=======================================================================
++|Setting|Replaced by
++| ssl | <<plugins-{type}s-{plugin}-ssl_enabled>>
+ |=======================================================================
+
```
Comment (karenzone) [inline docs/index.asciidoc]:
```suggestion
[id="plugins-{type}s-{plugin}-obsolete-options"]
==== Elasticsearch Output Obsolete Configuration Options

WARNING: As of version `2.0.0` of this plugin, some configuration options have been replaced.
The plugin will fail to start if it contains any of these obsolete options.

[cols="<,<",options="header",]
 |=======================================================================
|Setting|Replaced by
| ssl | <<plugins-{type}s-{plugin}-ssl_enabled>>
 |=======================================================================

```

Comment (karenzone) [reply docs/index.asciidoc]: Remove `+` signs carried over from copy/paste

**Review by karenzone — COMMENTED** [review]
On or around line 161, add note/link wrt deprecated settings that have been removed: 
"NOTE: As of version `2.0.0` of this plugin, a previously deprecated SSL setting has been removed. 
Please check out <<plugins-{type}s-{plugin}-obsolete-options>> for details." 

---
I tweaked the content a bit from the standard to make more sense with only one setting deleted. 


ALSO, please check out inline suggestion.

**Comment by karenzone** [comment]
Please also pick up initial suggestion noted in: https://github.com/logstash-plugins/logstash-input-elastic_serverless_forwarder/pull/11#pullrequestreview-2520891882

GitHub doesn't allow suggestions to sections without changes.  They're easy to miss.

**Comment by donoghuc** [comment]
Sorry, yeah i had an isssue where i committed your suggestion, then when i tried to pull it down locally i ended up on a different local index. I thought i had force pushed it but it was the wrong source<->dest. I think i've got it cleaned up now. Monday problems 😅

**Review by karenzone — APPROVED** [review]
LGTM

**Pull request merged**